--- src/gui/sdlmain.cpp.str	2009-05-28 09:45:43.000000000 +0200
+++ src/gui/sdlmain.cpp	2009-05-28 09:45:56.000000000 +0200
@@ -1526,7 +1526,7 @@
 	if ( !sdl.inited && SDL_Init(SDL_INIT_VIDEO|SDL_INIT_NOPARACHUTE) < 0 ) textonly = true;
 	sdl.inited = true;
 #endif
-	printf(message);
+	printf("%s",message);
 	if(textonly) return;
 	if(!sdl.surface) sdl.surface = SDL_SetVideoMode(640,400,0,0);
 	if(!sdl.surface) return;
